rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,432 @@
+
+## Definitions
+
+**Users of the DCP** include: Computational and non-computational consumers of the DCP, scientific data contributors to the DCP, and methods and application developers.
+
+**Project Leads**, **Product Owners**, and **User Experience** are members of the PM team as described in its [charter](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/PM/charter.md). Individual Project Leads and Product Owners are identified in [component charters](https://github.com/HumanCellAtlas/dcp-community/tree/master/charters).
+
+**Technical Leads** are core members of the Architecture Team as described in its  [charter](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Architecture/charter.md). Individual Technical Leads are identified in [component charters](https://github.com/HumanCellAtlas/dcp-community/tree/master/charters).
+
+**Themes** represent a discrete user or stakeholder need that delivers on the long term vision of the HCA. A Theme motivates a collection of work performed by the DCP.
+
+**Roadmap Objectives** are the concrete elements of an overarching Theme.
+
+**Product Roadmaps** identify and express user needs modeled as high-level Themes that capture user priorities. Roadmaps align the DCP community around a common set of Themes and get scientists excited about future releases.
+
+**Science Roadmaps** communicate a high-level vision of the goals of the single-cell community and a trajectory of progress towards those goals. It identifies both use cases for the constructed Atlas and open scientific questions that must be solved to build it.
+
+**[User Stories](http://agiledictionary.com/277/user-story/)** are requirements "stated as a sentence or two of plain English. A user story is often expressed from the user’s point of view, and describes a unit of desired functionality."
+
+**[Epics](http://agiledictionary.com/309/epic/)** 
+are a collection of User Stories required to complete an objective.
+
+**[Spikes](http://agiledictionary.com/209/spike/)** are adopted from eXtreme Programming for activities that require research or design before estimation and final implementation can occur.
+- [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+**Milestones** represent a four week timeline/duration in a Release.
+
+**Releases** are composed of three milestones and occur on a quarterly cadence.
+
+## Detailed Design for Roadmaps
+
+The Product Roadmap creates a shared understanding across the DCP community and Users of the DCP for investments and direction for the next year. The Roadmap process MUST occur on an annual basis. 
+
+Themes and Objectives from the Roadmap are the primary source of priorities for planning Releases each quarter.
```

Comment:
It is not clear to me from this document how Themes are created. To me, they should be following real user problems/needs, and as such be part of the Strategy, combined with real life evidence. 

Happy to be pointed to somewhere where this is clarified.